torch/csrc/jit/interpreter.cpp

```diff
@@ -496,10 +502,118 @@ struct CodeImpl {
     }
   }
 
-  void emitOperator(Node* node) {
-    emitLoadInputs(node->inputs());
+  void pushArguments(
+      Value* val,
+      at::ArrayRef<Value*> inputs,
+      std::vector<Value*>& stack,
+      std::unordered_set<Node*>& seen) {
+    seen.insert(val->node());
+    stack.push_back(val);
+    GRAPH_DEBUG("pushing second time ", val->debugName());
+    // arguments in reverse order
+    for (int i = inputs.size() - 1; i >= 0; i--) {
+      GRAPH_DEBUG("pushing ", inputs.at(i)->debugName());
+      stack.push_back(inputs.at(i));
+    }
+  }
+
+  void emitUseInline(Value* start) {
```

Comment:
This is too much code duplication. Probably requires a refactor so that there is only a single switch statement across operator types.